README.md

```diff
@@ -32,11 +32,12 @@ This tool can be used in standalone as a JAR executable (with the command line)
 #### Get help
 Use `java -jar cnesreport.jar -h` to get the following help about cnesreport:
 ````
-usage: java -jar cnesreport.jar [-a <arg>] [-c] [-d <arg>] [-e] [-h] [-l <arg>] [-o <arg>] [-p <arg>] [-r <arg>]
+usage: java -jar cnesreport.jar [-a <arg>] [-b <arg>] [-c] [-d <arg>] [-e] [-h] [-l <arg>] [-o <arg>] [-p <arg>] [-r <arg>]
        [-s <arg>] [-t <arg>] [-v] [-w] [-x <arg>]
 Generate editable reports for SonarQube projects.
 
  -a,--author <arg>                 Name of the report writer.
+ -b,--branch <arg>                 Branch of the targeted project. Requires Developer Edition or sonarqube-community-branch-plugin.
```

Comment:
Could you please write what the default value is ?
Just like the "date" parameter for example.

Reply:
Default is nothing, which means *do not use branches*. How do you think, we should call this default value? `(defaults to using no branches)`?

Reply:
I asked because I thought that maybe changing the default to master would be useful.
It may be useless in the end, I agree.

Reply:
Behaviour:
**SonarQube with branches-support**
`-b` is set --> the branch of the project (`-p`) is used
`-b` is not set --> the *main branch* of the project (`-p`) is used (mostly `master` but can be changed in SQ)
**SonarQube without branches-support**
`-b` is set --> undefined behaviour
`-b` is not set --> the project (`-p`) is used

Reply:
```suggestion
 -b,--branch <arg>                 Branch of the targeted project. Requires Developer Edition or sonarqube-community-branch-plugin. Default: usage of main branch.
```

Reply:
Thanks, looks good to me.

Reply:
Could you please commit the suggested changes ?  
Then I will validate the pull request.